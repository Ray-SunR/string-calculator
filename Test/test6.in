f a
s b a a
la
wa
lb
wb
ea
eb

n a abc
t b a 
wa
la
lb
wb

ea
eb

n a abc
m b 1 a
lb
wb
la
wa
p c a 1
lc
wc

ea
eb
ec

n a 12345
t b a abc
lb
wb
ea
eb

n a 123abc
la
wa
ea

n a 123abc
la
wa
ea
q
