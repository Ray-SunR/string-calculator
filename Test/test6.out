0

0

3
abc
5
abcwa
3
abc
3
abc
3
abc
8
12345abc
6
123abc
